script/bootstrap

```diff
@@ -12,9 +12,16 @@
 
 set -o errexit
 
+if [ ! $(whoami) == "root" ]; then
+  echo "You need superuser privileges to continue."
+fi
+
 if [ -f "/etc/os-release" ]; then
   . /etc/os-release
 fi
+if [ -n "$ID" ]; then
```

Comment:
This overwrites ID_LIKE if ID exists.  This breaks building on many other distros. I think the logic you want is if ID_LIKE is not defined then set it to ID...

Reply:
There are distros that do not have ID_LIKE and there are distros that do not have the ID defined. Someone chose to use ID_LIKE.

Reply:
Look at this: https://manpages.ubuntu.com/manpages/focal/man5/os-release.5.html

I believe that the default variable could be ID.

Reply:
The key thing about ID Like:

> It should list identifiers of operating systems that are closely related to the local operating system in regards to **packaging** and programming interfaces
           
What the script is trying to figure out is if we should be using apt or yum packages. If we used ID we would need to have a coverage of a lot more OS IDs... 

Ultimately if you just want to add support for Bullseye and not a general solution, I'm ok with it, but it can't break builds on other distros.  The current PR breaks CentOS